tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/util/Wait.java

```diff
@@ -41,13 +41,13 @@ public static boolean waitFor(final Condition condition, final long duration) th
 
    public static boolean waitFor(final Condition condition,
                                  final long duration,
-                                 final int sleepMillis) throws Exception {
```

Comment:
It's used by org.apache.activemq.artemis.tests.integration.proton.ProtonTestForHeader#testSimpleBytes.